ensembl/conf/json/macaca_fascicularis_vcf.json

```
@@ -0,0 +1,17 @@
+{
+  "collections": [
+    {
+      "id": "crab_macaque_EVA",
+      "source_name": "EVA",
+      "description": "Variants from EVA",
+      "species": "macaca_fascicularis",
+      "assembly": "Macaca_fascicularis_6.0",
+      "type": "remote",
+      "use_as_source" : 1,
+      "filename_template" : "http://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
```

Use HTTPS instead of HTTP.

```suggestion
      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
```